language/move-bytecode-verifier/src/verifier.rs

```diff
@@ -83,3 +92,37 @@ pub fn verify_script_with_config(config: &VerifierConfig, script: &CompiledScrip
     CodeUnitVerifier::verify_script(config, script)?;
     script_signature::verify_script(script, no_additional_script_signature_checks)
 }
+
+pub fn verify_script_with_config(config: &VerifierConfig, script: &CompiledScript) -> VMResult<()> {
+    if !config.use_separate_thread {
+        return verify_script_with_config_impl(config, script);
+    }
+    thread::scope(|s| {
```

Comment:
Even though the process doesn't abort when a thread panics, this does not make this safe.

Reply:
I suppose you are concerning memory leaks?